package.json

```diff
@@ -40,32 +40,25 @@
     "node": ">=4.5"
   },
   "devDependencies": {
-    "autocannon": "^0.16.5",
```

Comment:
Hi! Thank you for contributing :)

Can you restore this dependency?
It is needed by our [benchmark](https://github.com/cagataycali/fastify/blob/0390a042827c77477291a0b642c3b4a7dcefaaa0/package.json#L13) script :D

Reply:
We should remove that script. Fastify has benchmarks in separated repository now

Reply:
Nope, we are using it to test `master` against the `some-feature` branch to see if there are performances regression ;)